dissect/target/plugins/apps/webservers/apache.py

```diff
@@ -32,43 +128,76 @@ def infer_log_format(line: str) -> Optional[LogFormat]:
         vhost_combined = 'example.com:80 1.2.3.4 - - [19/Dec/2022:17:25:40 +0100] "GET / HTTP/1.1" 200 312 "-"
                           "Mozilla/5.0 (Windows NT 10.0; Win64; x64); AppleWebKit/537.36 (KHTML, like Gecko)
                           Chrome/108.0.0.0 Safari/537.36"'
-    """
 
-    first_part = line.split(" ")[0]
+    Two logformats encountered on Citrix Netscalers:
+        combined_resptime_with_citrix_hdrs  = '1.2.3.4 -> 192.168.4.20 - - [19/Dec/2022:17:25:12 +0100] [12311]
+                                               "GET / HTTP/1.1" 200 712 "-" "Mozilla/5.0
+                                               (Windows NT 10.0; Win64; x64); AppleWebKit/537.36 (KHTML, like Gecko)
+                                               Chrome/108.0.0.0 Safari/537.36" "Time: 11264 microsecs"'
+
+
+        combined_resptime                   = '127.0.0.1 - - [19/Dec/2022:17:25:12 +0100] [69195] "GET / HTTP/1.1" 200
+                                               18705 "-" "curl/7.78.0" "Time: 49835 microsecs"'
+
+    """
+    splitted_line = line.split(" ")
```

Comment:
You can just use `split()` here, since it splits on spaces by default (and I think you don't mind the fact that multiple consecutive whitespaces are seen as one separator ;)) -> https://docs.python.org/3/library/stdtypes.html#str.split
```suggestion
    splitted_line = line.split()
```